db/controllers/controller.js

```diff
@@ -0,0 +1,25 @@
+const db = require("../connection");
+
+const fetchTopicsFromDatabase = async () => {
+  try {
+    const queryResult = await db.query("SELECT * FROM topics;");
+    return queryResult.rows;
+  } catch (error) {
+    throw new Error("Error getting topics: " + error.message);
+  }
+};
+
+const getTopics = async (req, res, next) => {
+  try {
+    const topics = await fetchTopicsFromDatabase();
+    if (topics.length === 0) {
```

Comment:
This check here is unnecessary at the moment as our only error is bad URLs which will be handled in app.js